packages/base/src/3dview/mainview.tsx

```diff
@@ -117,6 +117,10 @@ export class MainView extends React.Component<IProps, IStates> {
     this.addContextMenu();
     this._mainViewModel.initWorker();
     this._mainViewModel.initSignal();
+    window.addEventListener('nodeClick', (e: Event) => {
```

Comment:
This will also listen to events coming from other CAD documents.

Reply:
Adding my 2 cents, what Trung means here is that if you have two docs opened, the two mains views will update when clicking in the left panel. This is not really what you want.

https://github.com/user-attachments/assets/28fe02ef-ded0-4a0f-93e6-f73b9057781c